bindings/rust/s2n-tls/benches/handshake.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use s2n_tls::testing::s2n_tls::tests::{build_config, s2n_tls_pair};
```

Comment:
The `tests` modules should not be public. If you want to use those functions they should be promoted to the parent module.

Reply:
done; lmk if there are some other examples/patterns to  read up on why this layout....